observatory-platform/requirements.txt

```diff
@@ -26,6 +26,7 @@ cerberus>=1.3.4,<2
 Jinja2>=2.10.1,<3
 stringcase>=1.2.0,<2
 pyhcl>=0.4.4,<1
+Sphinx==4.1.*
```

Comment:
It would be good to make the requirements definition for Sphinx looser, so that it doesn't cause problems for users of the observatory-platform, e.g. `Sphinx>=4,<5`. We should also change the Sphinx requirements in docs/requirements.txt to the same.